wiki/History.md

```diff
@@ -11,3 +11,4 @@ As far as I can tell (and as far as I was able to remember), this is the chain o
 * 2016 this became a product under the company name [Symless](https://web.archive.org/web/20160630194628/http://symless.com/synergy/) with Synergy as a product with binaries being sold.
 * In 2018, [Barrier](https://github.com/debauchee/barrier) made [the first releases](https://github.com/debauchee/barrier/releases/tag/v2.0.0-RC1) as a fork from symless/synergy
 * In 2022, [Input-Leap](https://github.com/input-leap/input-leap) was forked from Barrier
+* In 2024, the original open source project, Synergy, got renamed to [Deskflow](https://github.com/deskflow/deskflow). The name Synergy is now only used for the commercial offering which uses the Synergy/Deskflow code as a library. In the process, they also added a [history document](https://github.com/deskflow/deskflow/wiki/History) to their wiki.
```

Comment:
It's not a library. It's an upstream base that Synergy is built on. I would also avoid the 'Synergy/Deskflow' naming - as it's clarified at the beginning, Deskflow should be referred to as 'Deskflow' - not 'Synergy/Deskflow'. It may confuse people.

Reply:
I should also add that Deskflow can be used standalone.